exercises/concept/vec-macro/.docs/instructions.md

```diff
@@ -0,0 +1,20 @@
+You are going to initialize empty buffers and list the first five numbers, or elements, of the Fibonacci sequence.
+
+The Fibonacci sequence is a set of numbers where the next element is the sum of the prior two.
```

Comment:
You probably also need to define the first two numbers for this definition to be complete :)